### PR TITLE
Add doc comment for dxOverlayOptions_position

### PR DESCRIPTION
**Pull request by AntonSermyazhko**

### CODE REVIEW

**Review by RomanTsukanov — DISMISSED** [review]

**Comment by San4es** [comment]
Clarification:

The `position` option accepts 'left', 'top', 'right', 'bottom' and 'center' starting with [this PR](https://github.com/DevExpress/DevExtreme/pull/2601).

`Enums.Position` has no 'center' item:
https://github.com/DevExpress/DevExtreme/blob/17_2/js/docEnums.js#L137-L140

**Comment by San4es** [comment]
I suggest creating a separate enum (e.g. OverlayPosition) with "center", "left", "right", "top", "bottom" items.

@AntonSermyazhko @RomanTsukanov @lukyanovas @KsyushaM @vconst 
What do you think?

File: js/ui/overlay.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ var Overlay = Widget.inherit({
             /**
             * @name dxOverlayOptions_position
             * @publicName position
-            * @type string|positionConfig|function
+            * @type Enums.Position|positionConfig|function
```
Comment (San4es) [inline js/ui/overlay.js:184]: This enum has no "center" item.

**Review by San4es — CHANGES_REQUESTED** [review]

**Comment by RomanTsukanov** [comment]
@San4es seems like a good idea

**Comment by San4es** [comment]
@AntonSermyazhko @vconst @RomanTsukanov 
FYI: https://github.com/DevExpress/DevExtreme/pull/3111

Looks like we need to close this PR :)

**Pull request closed without merging**